robyn/__init__.py

```diff
@@ -302,110 +310,193 @@ def inner(handler):
 
         return inner
 
-    def get(self, endpoint: str, const: bool = False, auth_required: bool = False):
+    def get(
+        self,
+        endpoint: str,
+        const: bool = False,
+        auth_required: bool = False,
+        openapi_tags: list = ["get"],
+    ):
         """
         The @app.get decorator to add a route with the GET method
 
-        :param endpoint str: endpoint to server the route
+        :param endpoint str: endpoint for the route added
+        :param const bool: represents if the handler is a const function or not
+        :param auth_required bool: represents if the route needs authentication or not
+        :param openapi_tags: tags -- for grouping of endpoints in the openapi spec
         """
 
         def inner(handler):
+            self.openapi.add_openapi_path_obj("get", endpoint, inspect.getdoc(handler), openapi_tags, inspect.signature(handler))
```

Comment:
one question what happens if there is no docstring? i.e in `inspect.getdoc`?

Reply:
prints "None" -- I will add a fallback value.

Reply:
@VishnuSanal , this is unresolved right?

Reply:
no, this is fixed. the summary falls back to "No summary provided".